From 5ff0e56241355f9d02ae43e8201f4ceaa65ae204 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Thu, 14 Jun 2018 21:52:15 +0300
Subject: [PATCH] FIXME: NET irq storm

We avoid eth0 up since there is an IRQ storm
Which hangs machine
---

diff --git a/board/habanalabs/dali/post-build.sh b/board/habanalabs/dali/post-build.sh
index 55831c1..f2ef87c 100755
--- a/board/habanalabs/dali/post-build.sh
+++ b/board/habanalabs/dali/post-build.sh
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
 
 pushd ${TARGET_DIR}/etc/init.d/
-	if [ "${DALI_TARGET}" == "pldm" ]; then
+	if [ "${DALI_TARGET}" == "pldm" ] || [ "${DALI_TARGET}" == "asic" ]; then
 		if [ -e S40network ]; then
 			mv S40network .S40network
 		fi
-	elif [ "${DALI_TARGET}" == "onix" ] || [ "${DALI_TARGET}" == "asic" ]; then
+	elif [ "${DALI_TARGET}" == "onix" ]; then
 		if [ -e .S40network ]; then
 			mv .S40network S40network
 		fi
